04_numeronym/paraganz/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronyms(vals ...string) []string {
+	var arr = make([]string, len(vals))
+	for i := 0; i < len(vals); i++ {
+		val := vals[i]
+		if len(vals[i]) > 4 {
```

Comment:
hmm how about `abcd`, shouldn't that be `a2d` ?

Reply:
Done